Source/Scene/PointCloud3DTileContent.js

```diff
@@ -520,6 +532,16 @@ define([
         if (hasColors) {
             if (isTranslucent) {
                 vs += 'attribute vec4 a_color; \n';
+            } else if (isYCoCg || isRGB565) {
+                vs += 'attribute float a_color; \n' +
+                      'const float SHIFT_RIGHT_11 = 1.0 / 2048.0; \n' +
```

Comment:
@lilleyse @bagnell should we introduce built-in `czm_` GLSL functions for shifts to basically extract a set of bits from a float?